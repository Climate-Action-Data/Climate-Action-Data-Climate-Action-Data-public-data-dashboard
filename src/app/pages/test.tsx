'use client'
import { type NextPage } from "next";

const Test: NextPage = () => {
    return (
        <div>
            <h1>Hello</h1>
            <p>data</p>
        </div>
    )
}

export default Test