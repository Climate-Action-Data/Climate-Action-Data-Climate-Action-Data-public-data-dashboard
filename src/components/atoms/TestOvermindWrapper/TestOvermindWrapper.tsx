import { createOvermind } from 'overmind'
import { Provider } from 'overmind-react'
import { config } from '@/overmind'
import { PropsWithChildren } from 'react'

const overmind = createOvermind(config)

export const TestOvermindWrapper = (props: PropsWithChildren) => {
  return <Provider value={overmind}>{props.children}</Provider>
}
