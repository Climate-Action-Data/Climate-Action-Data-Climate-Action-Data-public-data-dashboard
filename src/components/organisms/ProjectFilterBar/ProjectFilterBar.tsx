import { FC, useEffect } from 'react'
import { useTranslation } from 'react-i18next'
import { Stack, StackDivider } from '@chakra-ui/react'

import { useActions, useAppState, useEffects } from '@/overmind'
import AutoCompleteCheckbox from '@/components/molecules/AutoCompleteCheckbox/AutoCompleteCheckbox'
import CreditingPeriodFilter from '@/components/molecules/CreditingPeriodFilter/CreditingPeriodFilter'
import { KYOTO_PROTOCOL_START_DATE } from '@/@types/CarbonStandards'
import { DatesFilter } from '@/@types/ProjectSearchFilterValues'
import FilterBarWrapper from '@/components/molecules/FilterBarWarpper/FilterBarWrapper'

interface ProjectFilterBarProps {
  isResultsPage?: boolean
}

const ProjectFilterBar: FC<ProjectFilterBarProps> = (props) => {
  const { isResultsPage } = props
  const {
    allSearchFilterValues: { searchFilterValues, isEmpty },
    selectedProjectSearchFilterValues: { searchFilterValues: selectedSearchFilters },
  } = useAppState().searchFilters

  const { setProjectCountriesFilter, setProjectMethodologiesFilter, setProjectSectorsFilter, setProjectStandardsFilter, setProjectCreditingPeriodFilter } =
    useActions().searchFilters
  const { getGovernanceData } = useEffects().searchFilters
  const { transformGovernanceDataToSearchFilterData } = useActions().searchFilters
  const { t } = useTranslation(`search`)

  useEffect(() => {
    if (isEmpty) {
      getGovernanceData().then((result) => {
        transformGovernanceDataToSearchFilterData(result)
      })
    }
  }, [])

  const handleProjectStatusFilter = (value: string[]) => {
    setProjectStandardsFilter(value)
  }

  const handleSetProjectMethodologiesFilter = (value: string[]) => {
    setProjectMethodologiesFilter(value)
  }

  const handleSetProjectSectorsFilter = (value: string[]) => {
    setProjectSectorsFilter(value)
  }

  const handleSetProjectCountriesFilter = (value: string[]) => {
    setProjectCountriesFilter(value)
  }

  const handleSetProjectCreditingPeriodFilter = (value: DatesFilter) => {
    setProjectCreditingPeriodFilter(value)
  }

  return (
    <FilterBarWrapper isResultsPage={isResultsPage}>
      <Stack direction={[`column`, null, null, `row`]}>
        <Stack direction={[`column`, null, null, `row`]} divider={<StackDivider borderColor="lightGray.400" />} width="100%">
          <AutoCompleteCheckbox
            label={t(`filterBar.standard`)}
            options={searchFilterValues.projectStatus}
            selectedFilters={selectedSearchFilters.projectStatus}
            noOfSelectedFilters={selectedSearchFilters.projectStatus.length}
            applyFilters={handleProjectStatusFilter}
            isResultsPage={isResultsPage}
          />
          <AutoCompleteCheckbox
            label={t(`filterBar.methodology`)}
            options={searchFilterValues.methodologies}
            selectedFilters={selectedSearchFilters.methodologies}
            noOfSelectedFilters={selectedSearchFilters.methodologies.length}
            applyFilters={handleSetProjectMethodologiesFilter}
            isResultsPage={isResultsPage}
          />
          <AutoCompleteCheckbox
            label={t(`filterBar.sector`)}
            options={searchFilterValues.sectors}
            selectedFilters={selectedSearchFilters.sectors}
            noOfSelectedFilters={selectedSearchFilters.sectors.length}
            applyFilters={handleSetProjectSectorsFilter}
            isResultsPage={isResultsPage}
          />
          <AutoCompleteCheckbox
            label={t(`filterBar.country`)}
            options={searchFilterValues.countries}
            selectedFilters={selectedSearchFilters.countries}
            noOfSelectedFilters={selectedSearchFilters.countries.length}
            applyFilters={handleSetProjectCountriesFilter}
            isResultsPage={isResultsPage}
          />
          <CreditingPeriodFilter
            label={t(`filterBar.creditingPeriod`)}
            dateFilter={selectedSearchFilters.creditingPeriod}
            earliestDate={KYOTO_PROTOCOL_START_DATE}
            applyFilters={handleSetProjectCreditingPeriodFilter}
            isResultsPage={isResultsPage}
          />
        </Stack>
      </Stack>
    </FilterBarWrapper>
  )
}

export default ProjectFilterBar
