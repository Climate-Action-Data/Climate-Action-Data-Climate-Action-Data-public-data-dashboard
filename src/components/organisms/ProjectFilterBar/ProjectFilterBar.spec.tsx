import { TestOvermindWrapper } from '@/test/TestOvermindWrapper'
import { render, screen, fireEvent } from '@testing-library/react'
import { MockData } from '@/test/TestOvermindMockData'
import ProjectFilterBar from '@/components/organisms/ProjectFilterBar/ProjectFilterBar'
import { TestRouter } from '@/test/TestRouter'

const mockPush = jest.fn()
jest.mock(`next/navigation`, () => ({
  ...jest.requireActual(`next/navigation`),
  useRouter: () => ({ push: mockPush, query: { query: `testId` } }),
  useSearchParams: () => ({ get: () => `testParams` }),
}))

const push = jest.fn()

it(`renders correctly with no data`, () => {
  const { container } = render(
    <TestOvermindWrapper>
      <ProjectFilterBar />
    </TestOvermindWrapper>,
  )
  expect(container).toMatchSnapshot()
})

it(`renders correctly with loaded data`, () => {
  const { container } = render(
    <TestOvermindWrapper searchFilters={MockData.SEARCH_FILTER_VALUES}>
      <ProjectFilterBar />
    </TestOvermindWrapper>,
  )
  expect(container).toMatchSnapshot()
})

it(`renders correctly with loaded data and search click`, () => {
  const { container } = render(
    <TestOvermindWrapper searchFilters={MockData.SEARCH_FILTER_VALUES}>
      <TestRouter router={{ push }}>
        <ProjectFilterBar />
      </TestRouter>
    </TestOvermindWrapper>,
  )
  const button = screen.getByTestId(`search-button`)
  fireEvent.click(button)
  expect(container).toMatchSnapshot()
})
