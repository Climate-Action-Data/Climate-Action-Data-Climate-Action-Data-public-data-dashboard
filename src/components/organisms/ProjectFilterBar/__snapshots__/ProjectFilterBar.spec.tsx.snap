// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly with loaded data 1`] = `
<div>
  <div
    class="css-1mjj88o"
  >
    <div
      class="chakra-stack css-18mjik"
    >
      <div
        class="chakra-stack css-17z0ye3"
      >
        <button
          aria-controls="popover-content-:rb:"
          aria-expanded="false"
          aria-haspopup="dialog"
          class="chakra-button css-4xx2wk"
          id="popover-trigger-:rb:"
          type="button"
        >
          <div
            class="css-jmqhys"
          >
            Standard
            <div
              class="css-1gq8co7"
            />
            <div
              class="css-1vzx6oo"
            >
              <p
                class="chakra-text css-11iqvn7"
              >
                3
              </p>
            </div>
            <div
              class="css-17xejub"
            />
            <svg
              class="chakra-icon css-c0air1"
              focusable="false"
              viewBox="0 0 8 4"
            >
              <path
                d="M3.99911 3.9034L0.191406 0.0957031H7.80681L3.99911 3.9034Z"
                fill="currentColor"
              />
            </svg>
          </div>
        </button>
        <div
          class="chakra-popover__popper css-0"
          style="visibility: hidden; position: absolute; inset: 0 auto auto 0;"
        >
          <section
            class="chakra-popover__content css-gt8u74"
            id="popover-content-:rb:"
            role="dialog"
            style="transform-origin: var(--popper-transform-origin); opacity: 0; visibility: hidden; transform: scale(0.95) translateZ(0);"
            tabindex="-1"
          />
        </div>
        <div
          class="chakra-stack__divider css-1psh3p6"
        />
        <button
          aria-controls="popover-content-:rd:"
          aria-expanded="false"
          aria-haspopup="dialog"
          class="chakra-button css-4xx2wk"
          id="popover-trigger-:rd:"
          type="button"
        >
          <div
            class="css-jmqhys"
          >
            Methodology
            <div
              class="css-1gq8co7"
            />
            <div
              class="css-1vzx6oo"
            >
              <p
                class="chakra-text css-11iqvn7"
              >
                3
              </p>
            </div>
            <div
              class="css-17xejub"
            />
            <svg
              class="chakra-icon css-c0air1"
              focusable="false"
              viewBox="0 0 8 4"
            >
              <path
                d="M3.99911 3.9034L0.191406 0.0957031H7.80681L3.99911 3.9034Z"
                fill="currentColor"
              />
            </svg>
          </div>
        </button>
        <div
          class="chakra-popover__popper css-0"
          style="visibility: hidden; position: absolute; inset: 0 auto auto 0;"
        >
          <section
            class="chakra-popover__content css-gt8u74"
            id="popover-content-:rd:"
            role="dialog"
            style="transform-origin: var(--popper-transform-origin); opacity: 0; visibility: hidden; transform: scale(0.95) translateZ(0);"
            tabindex="-1"
          />
        </div>
        <div
          class="chakra-stack__divider css-1psh3p6"
        />
        <button
          aria-controls="popover-content-:rf:"
          aria-expanded="false"
          aria-haspopup="dialog"
          class="chakra-button css-4xx2wk"
          id="popover-trigger-:rf:"
          type="button"
        >
          <div
            class="css-jmqhys"
          >
            Sector
            <div
              class="css-1gq8co7"
            />
            <div
              class="css-1vzx6oo"
            >
              <p
                class="chakra-text css-11iqvn7"
              >
                3
              </p>
            </div>
            <div
              class="css-17xejub"
            />
            <svg
              class="chakra-icon css-c0air1"
              focusable="false"
              viewBox="0 0 8 4"
            >
              <path
                d="M3.99911 3.9034L0.191406 0.0957031H7.80681L3.99911 3.9034Z"
                fill="currentColor"
              />
            </svg>
          </div>
        </button>
        <div
          class="chakra-popover__popper css-0"
          style="visibility: hidden; position: absolute; inset: 0 auto auto 0;"
        >
          <section
            class="chakra-popover__content css-gt8u74"
            id="popover-content-:rf:"
            role="dialog"
            style="transform-origin: var(--popper-transform-origin); opacity: 0; visibility: hidden; transform: scale(0.95) translateZ(0);"
            tabindex="-1"
          />
        </div>
        <div
          class="chakra-stack__divider css-1psh3p6"
        />
        <button
          aria-controls="popover-content-:rh:"
          aria-expanded="false"
          aria-haspopup="dialog"
          class="chakra-button css-4xx2wk"
          id="popover-trigger-:rh:"
          type="button"
        >
          <div
            class="css-jmqhys"
          >
            Country
            <div
              class="css-1gq8co7"
            />
            <div
              class="css-1vzx6oo"
            >
              <p
                class="chakra-text css-11iqvn7"
              >
                3
              </p>
            </div>
            <div
              class="css-17xejub"
            />
            <svg
              class="chakra-icon css-c0air1"
              focusable="false"
              viewBox="0 0 8 4"
            >
              <path
                d="M3.99911 3.9034L0.191406 0.0957031H7.80681L3.99911 3.9034Z"
                fill="currentColor"
              />
            </svg>
          </div>
        </button>
        <div
          class="chakra-popover__popper css-0"
          style="visibility: hidden; position: absolute; inset: 0 auto auto 0;"
        >
          <section
            class="chakra-popover__content css-gt8u74"
            id="popover-content-:rh:"
            role="dialog"
            style="transform-origin: var(--popper-transform-origin); opacity: 0; visibility: hidden; transform: scale(0.95) translateZ(0);"
            tabindex="-1"
          />
        </div>
        <div
          class="chakra-stack__divider css-1psh3p6"
        />
        <button
          aria-controls="popover-content-:rj:"
          aria-expanded="false"
          aria-haspopup="dialog"
          class="chakra-button css-4xx2wk"
          id="popover-trigger-:rj:"
          type="button"
        >
          <div
            class="css-jmqhys"
          >
            <div
              class="css-k008qs"
            >
              Crediting Period
            </div>
            <div
              class="css-17xejub"
            />
            <svg
              class="chakra-icon css-c0air1"
              focusable="false"
              viewBox="0 0 8 4"
            >
              <path
                d="M3.99911 3.9034L0.191406 0.0957031H7.80681L3.99911 3.9034Z"
                fill="currentColor"
              />
            </svg>
          </div>
        </button>
        <div
          class="chakra-popover__popper css-0"
          style="visibility: hidden; position: absolute; inset: 0 auto auto 0;"
        >
          <section
            class="chakra-popover__content css-1j9y0nx"
            id="popover-content-:rj:"
            role="dialog"
            style="transform-origin: var(--popper-transform-origin); opacity: 0; visibility: hidden; transform: scale(0.95) translateZ(0);"
            tabindex="-1"
          />
        </div>
      </div>
      <div
        class="css-k008qs"
      >
        <div
          class="css-1lkwiwl"
        />
        <a
          class="chakra-link css-183olx0"
          href="/search/projects"
        >
          <div
            class="css-70qvj9"
          >
            Search
            <svg
              class="chakra-icon css-1hrwg97"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M19.1623 20.2239L13.4084 14.4701C12.9084 14.8829 12.3334 15.206 11.6834 15.4393C11.0334 15.6726 10.361 15.7893 9.66613 15.7893C7.95696 15.7893 6.51045 15.1975 5.32658 14.0139C4.14271 12.8304 3.55078 11.3842 3.55078 9.67552C3.55078 7.96678 4.14256 6.52011 5.32613 5.33549C6.5097 4.15089 7.95585 3.55859 9.66458 3.55859C11.3733 3.55859 12.82 4.15053 14.0046 5.33439C15.1892 6.51826 15.7815 7.96478 15.7815 9.67394C15.7815 10.388 15.6616 11.0701 15.4219 11.7201C15.1821 12.3701 14.8623 12.9355 14.4623 13.4162L20.2411 19.1951C20.3796 19.3309 20.4488 19.4975 20.4488 19.695C20.4488 19.8925 20.3706 20.0695 20.214 20.226C20.0705 20.3695 19.8956 20.4412 19.6892 20.4412C19.4828 20.4412 19.3072 20.3688 19.1623 20.2239ZM9.66613 14.2893C10.9546 14.2893 12.0459 13.8422 12.9402 12.948C13.8344 12.0538 14.2815 10.9624 14.2815 9.67394C14.2815 8.38548 13.8344 7.29413 12.9402 6.39989C12.0459 5.50566 10.9546 5.05854 9.66613 5.05854C8.37766 5.05854 7.28631 5.50566 6.39208 6.39989C5.49786 7.29413 5.05076 8.38548 5.05076 9.67394C5.05076 10.9624 5.49786 12.0538 6.39208 12.948C7.28631 13.8422 8.37766 14.2893 9.66613 14.2893Z"
                fill="currentColor"
              />
            </svg>
          </div>
        </a>
      </div>
    </div>
  </div>
</div>
`;

exports[`renders correctly with no data 1`] = `
<div>
  <div
    class="css-1mjj88o"
  >
    <div
      class="chakra-stack css-18mjik"
    >
      <div
        class="chakra-stack css-17z0ye3"
      >
        <button
          aria-controls="popover-content-:r1:"
          aria-expanded="false"
          aria-haspopup="dialog"
          class="chakra-button css-4xx2wk"
          id="popover-trigger-:r1:"
          type="button"
        >
          <div
            class="css-jmqhys"
          >
            Standard
            <div
              class="css-1gq8co7"
            />
            <div
              class="css-17xejub"
            />
            <svg
              class="chakra-icon css-c0air1"
              focusable="false"
              viewBox="0 0 8 4"
            >
              <path
                d="M3.99911 3.9034L0.191406 0.0957031H7.80681L3.99911 3.9034Z"
                fill="currentColor"
              />
            </svg>
          </div>
        </button>
        <div
          class="chakra-popover__popper css-0"
          style="visibility: hidden; position: absolute; inset: 0 auto auto 0;"
        >
          <section
            class="chakra-popover__content css-gt8u74"
            id="popover-content-:r1:"
            role="dialog"
            style="transform-origin: var(--popper-transform-origin); opacity: 0; visibility: hidden; transform: scale(0.95) translateZ(0);"
            tabindex="-1"
          />
        </div>
        <div
          class="chakra-stack__divider css-1psh3p6"
        />
        <button
          aria-controls="popover-content-:r3:"
          aria-expanded="false"
          aria-haspopup="dialog"
          class="chakra-button css-4xx2wk"
          id="popover-trigger-:r3:"
          type="button"
        >
          <div
            class="css-jmqhys"
          >
            Methodology
            <div
              class="css-1gq8co7"
            />
            <div
              class="css-17xejub"
            />
            <svg
              class="chakra-icon css-c0air1"
              focusable="false"
              viewBox="0 0 8 4"
            >
              <path
                d="M3.99911 3.9034L0.191406 0.0957031H7.80681L3.99911 3.9034Z"
                fill="currentColor"
              />
            </svg>
          </div>
        </button>
        <div
          class="chakra-popover__popper css-0"
          style="visibility: hidden; position: absolute; inset: 0 auto auto 0;"
        >
          <section
            class="chakra-popover__content css-gt8u74"
            id="popover-content-:r3:"
            role="dialog"
            style="transform-origin: var(--popper-transform-origin); opacity: 0; visibility: hidden; transform: scale(0.95) translateZ(0);"
            tabindex="-1"
          />
        </div>
        <div
          class="chakra-stack__divider css-1psh3p6"
        />
        <button
          aria-controls="popover-content-:r5:"
          aria-expanded="false"
          aria-haspopup="dialog"
          class="chakra-button css-4xx2wk"
          id="popover-trigger-:r5:"
          type="button"
        >
          <div
            class="css-jmqhys"
          >
            Sector
            <div
              class="css-1gq8co7"
            />
            <div
              class="css-17xejub"
            />
            <svg
              class="chakra-icon css-c0air1"
              focusable="false"
              viewBox="0 0 8 4"
            >
              <path
                d="M3.99911 3.9034L0.191406 0.0957031H7.80681L3.99911 3.9034Z"
                fill="currentColor"
              />
            </svg>
          </div>
        </button>
        <div
          class="chakra-popover__popper css-0"
          style="visibility: hidden; position: absolute; inset: 0 auto auto 0;"
        >
          <section
            class="chakra-popover__content css-gt8u74"
            id="popover-content-:r5:"
            role="dialog"
            style="transform-origin: var(--popper-transform-origin); opacity: 0; visibility: hidden; transform: scale(0.95) translateZ(0);"
            tabindex="-1"
          />
        </div>
        <div
          class="chakra-stack__divider css-1psh3p6"
        />
        <button
          aria-controls="popover-content-:r7:"
          aria-expanded="false"
          aria-haspopup="dialog"
          class="chakra-button css-4xx2wk"
          id="popover-trigger-:r7:"
          type="button"
        >
          <div
            class="css-jmqhys"
          >
            Country
            <div
              class="css-1gq8co7"
            />
            <div
              class="css-17xejub"
            />
            <svg
              class="chakra-icon css-c0air1"
              focusable="false"
              viewBox="0 0 8 4"
            >
              <path
                d="M3.99911 3.9034L0.191406 0.0957031H7.80681L3.99911 3.9034Z"
                fill="currentColor"
              />
            </svg>
          </div>
        </button>
        <div
          class="chakra-popover__popper css-0"
          style="visibility: hidden; position: absolute; inset: 0 auto auto 0;"
        >
          <section
            class="chakra-popover__content css-gt8u74"
            id="popover-content-:r7:"
            role="dialog"
            style="transform-origin: var(--popper-transform-origin); opacity: 0; visibility: hidden; transform: scale(0.95) translateZ(0);"
            tabindex="-1"
          />
        </div>
        <div
          class="chakra-stack__divider css-1psh3p6"
        />
        <button
          aria-controls="popover-content-:r9:"
          aria-expanded="false"
          aria-haspopup="dialog"
          class="chakra-button css-4xx2wk"
          id="popover-trigger-:r9:"
          type="button"
        >
          <div
            class="css-jmqhys"
          >
            <div
              class="css-k008qs"
            >
              Crediting Period
            </div>
            <div
              class="css-17xejub"
            />
            <svg
              class="chakra-icon css-c0air1"
              focusable="false"
              viewBox="0 0 8 4"
            >
              <path
                d="M3.99911 3.9034L0.191406 0.0957031H7.80681L3.99911 3.9034Z"
                fill="currentColor"
              />
            </svg>
          </div>
        </button>
        <div
          class="chakra-popover__popper css-0"
          style="visibility: hidden; position: absolute; inset: 0 auto auto 0;"
        >
          <section
            class="chakra-popover__content css-1j9y0nx"
            id="popover-content-:r9:"
            role="dialog"
            style="transform-origin: var(--popper-transform-origin); opacity: 0; visibility: hidden; transform: scale(0.95) translateZ(0);"
            tabindex="-1"
          />
        </div>
      </div>
      <div
        class="css-k008qs"
      >
        <div
          class="css-1lkwiwl"
        />
        <a
          class="chakra-link css-183olx0"
          href="/search/projects"
        >
          <div
            class="css-70qvj9"
          >
            Search
            <svg
              class="chakra-icon css-1hrwg97"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M19.1623 20.2239L13.4084 14.4701C12.9084 14.8829 12.3334 15.206 11.6834 15.4393C11.0334 15.6726 10.361 15.7893 9.66613 15.7893C7.95696 15.7893 6.51045 15.1975 5.32658 14.0139C4.14271 12.8304 3.55078 11.3842 3.55078 9.67552C3.55078 7.96678 4.14256 6.52011 5.32613 5.33549C6.5097 4.15089 7.95585 3.55859 9.66458 3.55859C11.3733 3.55859 12.82 4.15053 14.0046 5.33439C15.1892 6.51826 15.7815 7.96478 15.7815 9.67394C15.7815 10.388 15.6616 11.0701 15.4219 11.7201C15.1821 12.3701 14.8623 12.9355 14.4623 13.4162L20.2411 19.1951C20.3796 19.3309 20.4488 19.4975 20.4488 19.695C20.4488 19.8925 20.3706 20.0695 20.214 20.226C20.0705 20.3695 19.8956 20.4412 19.6892 20.4412C19.4828 20.4412 19.3072 20.3688 19.1623 20.2239ZM9.66613 14.2893C10.9546 14.2893 12.0459 13.8422 12.9402 12.948C13.8344 12.0538 14.2815 10.9624 14.2815 9.67394C14.2815 8.38548 13.8344 7.29413 12.9402 6.39989C12.0459 5.50566 10.9546 5.05854 9.66613 5.05854C8.37766 5.05854 7.28631 5.50566 6.39208 6.39989C5.49786 7.29413 5.05076 8.38548 5.05076 9.67394C5.05076 10.9624 5.49786 12.0538 6.39208 12.948C7.28631 13.8422 8.37766 14.2893 9.66613 14.2893Z"
                fill="currentColor"
              />
            </svg>
          </div>
        </a>
      </div>
    </div>
  </div>
</div>
`;
